catalog/designers/maniackersdesign/bio.html

```diff
@@ -0,0 +1,4 @@
+<p>
+Maniackers Design is a design studio works in various media. Established in 1995, it's based in Takasaki, Gunma in Japan.  Representative is Masayuki Sato. Maniackers Design started creating & uploading open source fonts on the website in 1998.
+</p>
```

Comment:
Would say: "Maniackers Design is a design studio directed by Masayuki Sato. Established in 1995, they are based in Takasaki, Gunma in Japan. Maniackers Design has been creating open source fonts since 1998."

(and remove www so it is just "mksd.jp")